handlers.go

```diff
@@ -60,6 +61,37 @@ func AddBeer(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 }
 
 // AddBeerReview adds a new review for a beer
-func AddBeerReview(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
-	// TODO
+func AddBeerReview(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	w.Header().Set("Content-Type", "application/json")
+
+	ID, err := strconv.Atoi(ps.ByName("id"))
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(fmt.Sprintf("%s is not a valid Beer ID, it must be a number.", ps.ByName("id")))
+		return
```

Comment:
Other folks started using http.Error for returning erroneous responses, so mind changing yours to use it as well? 

E.g. 
```
http.Error(w, fmt.Sprintf("%s is not a valid Beer ID, it must be a number.", ps.ByName("id")), http.StatusBadRequest)
return
```

Thanks and sorry for the trouble! Will be nice to keep things consistent though :)